azure/durable_functions/models/DurableOrchestrationBindings.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, taskHubName: str, creationUrls: Dict[str, str],
         self._task_hub_name: str = taskHubName
         self._creation_urls: Dict[str, str] = creationUrls
         self._management_urls: Dict[str, str] = managementUrls
+        # TODO: we can remove this once we drop support for 1.x, this is always provided in 2.x
```

Comment:
small nit: clarify what "this" is so you delete the "Optional" not the whole property :)

Reply:
Done!